sda-commons-shared-asyncapi/src/test/java/org/sdase/commons/shared/asyncapi/Iso8601SerializerTest.java

```diff
@@ -0,0 +1,20 @@
+package org.sdase.commons.shared.asyncapi;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+class Iso8601SerializerTest {
```

Comment:
For asyncapi, this is already covered [here](https://github.com/SDA-SE/sda-dropwizard-commons/blob/39c053d/sda-commons-shared-asyncapi/src/test/java/org/sdase/commons/shared/asyncapi/jsonschema/AbstractJsonSchemaBuilderTest.java#L119). Imho we can save some test code here. I assume, `HouseCreated` and the `asyncapi_template.yaml` are also just added for this test?

If you want a dedicated test, I would prefer to test only the schema generator and not the whole async api.

Reply:
Oh, good to know. I will remove the class and files again! 👍